src/install-pnpm/run.ts

```diff
@@ -49,7 +49,7 @@ Otherwise, please specify the pnpm version in the action configuration.`)
     throw new Error(`No pnpm version is specified.
 Please specify it by one of the following ways:
   - in the GitHub Action config with the key "version"
-  - in the package.json with the key "packageManager" (See https://nodejs.org/api/corepack.html)`)
+  - in the package.json with the key "packageManager" ( See https://nodejs.org/api/corepack.html )`)
```

Comment:
Why is this link even necessary? It is not pnpm's responsibility to teach its user about the fundamental of Node.js.

```suggestion
  - in the package.json with the key "packageManager"`)
```

Reply:
If maintainers don't not consider it necessary, it can be removed. 

I think that this is a helpful info for users so it is better to keep.
FYI, same link exists in README .

Do I remove it?

Reply:
I created another PR(#69). Merge the one you chose and close the other.